src/ResourceManager/KeyVault/Commands.KeyVault/Commands/SetAzureKeyVaultAccessPolicy.cs

```diff
@@ -235,6 +367,22 @@ public class SetAzureKeyVaultAccessPolicy : KeyVaultManagementCmdletBase
             ParameterSetName = ByEmailAddress,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Specifies certificate operation permissions to grant to a user or service principal.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByObjectId,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies certificate operation permissions to grant to a user or service principal.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByServicePrincipalName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies certificate operation permissions to grant to a user or service principal.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByUserPrincipalName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies certificate operation permissions to grant to a user or service principal.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByEmailAddress,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies certificate operation permissions to grant to a user or service principal.")]
         [ValidateSet("get", "list", "delete", "create", "import", "update", "managecontacts", "getissuers", "listissuers", "setissuers", "deleteissuers", "manageissuers", "recover", "purge", "all")]
```

Comment:
As an aside, we wanted to deprecate 'all' as a permission for some time. Since this is a deprecating change, we should just remove it with this commit. If you'd rather not, that's perfectly fine - we'll follow up. Thanks.

Reply:
This is not a breaking change, but we should deprecate anything we would like to remove at the next breaking change release

Reply:
All occurrences of "All" as a permission should be removed. Thanks.

Reply:
Added to list of breaking changes - will be included in my next PR.